libs/search-widget/src/core/api.ts

```diff
@@ -257,7 +257,7 @@ export const saveEntities = (backup: EntityGroup[], newGroups: EntityGroup[]) =>
     const backupGroup = backup.find((g) => g.id === group.id);
     if (!!backupGroup && JSON.stringify(group.entities) !== JSON.stringify(backupGroup.entities)) {
       requestList.push(
-        writableKb.setEntitiesGroup(group.id, {
+        writableKb.createEntitiesGroup(group.id, {
```

Comment:
`saveEntities` and `saveEntitiesAnnotations` are obsolete and must be removed (`setLabels` too actually, even if it is not related to this PR).